### PR TITLE
Added Sauce Labs to Automated Browser Testing section

### PR DESCRIPTION
**Pull request by Marketionist**
**This pull request adds Sauce Labs to Automated Browser Testing section**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -158,6 +158,7 @@ Table of Contents
 ## Automated Browser Testing
 
   * [gridlastic.com](https://www.gridlastic.com/) — Selenium Grid testing with free plan up to 4 simultaneous selenium nodes/10 grid starts/4,000 test minutes/month
+  * [saucelabs.com](https://saucelabs.com/) — Cross browser testing, Selenium testing and mobile testing, [free for Open Source](https://saucelabs.com/open-source)
```
Comment (ripienaar) [inline README.md]: it's already listed in the section above, not sure its really adding enough value to list it twice, or did you intend to list it twice?

Comment (Marketionist) [inline README.md]: @ripienaar I believe it'll be better to move it from CI / CD to Automated Browser Testing - as it is primary purpose is much more about testing, not Continuous Integration or Continuous Delivery.

Will it be ok if I'll delete it from CI / CD section?

Comment (ripienaar) [inline README.md]: @Marketionist yup, that sounds good to me

**Comment by ripienaar** [comment]
Thanks @Marketionist

**Comment by Marketionist** [comment]
Thanks for reviewing @ripienaar

**Pull request merged**